app/assets/javascripts/games.js.coffee

```diff
@@ -8,6 +8,12 @@ $ ->
     channel.bind 'refresh_event', (data)->
       location.reload()
 
+  $('#myModal').on 'show.bs.modal', (e)->
+    userId = $(e.relatedTarget).data('playerid')
+    url = window.location.protocol + '//' + window.location.host + '/users/' + userId
+    $.get url, (response)->
+      $('.modal-body').html('<h4>' + response.name + ' has been registered since ' + response.playing_since + ' and has ' + response.total_wins + ' total wins.' + '</h4>')
```

Comment:
If the user opens and closes one modal and then opens another they will see the old data until the AJAX loads. Probably not noticeable on locahost but could be somewhat jarring on a slow connection. Maybe we could clear out the modal before we do the ajax call so at least they won't see wrong data?